src/pages/SignUp/services/SignUp.Hooks.js

```diff
@@ -21,7 +22,7 @@ export const useSignUp = () => {
           title: 'Sign up success',
           description: 'Check your mailbox!',
         });
-        navigate(paths.signin);
+        navigate(paths.home);
```

Comment:
this is also changes that shouldn't be here

Reply:
You can use this command to revert a file to a branch. In this case, main.
```
git checkout main -- src/pages/SignUp/services/SignUp.Hooks.js
```

```
git checkout <branchname> -- <filename>
```

Reply:
git checkout main -- src/pages/SignUp/services/SignUp.Hooks.js
I did, for next should I commit?

Reply:
Yeah when you run that command, it should revert that file back to what it is on `main` branch. Commit that change and push it up here.